docs/get-started/quickstarts/detect-people.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Detect People with a Webcam"
+linkTitle: "Detect people with a webcam"
+weight: 50
+type: "docs"
+tags: ["vision", "data", "services", "quickstart", "ml", "camera", "webcam"]
+no_list: true
+description: "Use Viam's machine learning capabilities to deploy a person detector ML model to your machine."
+images: ["/platform/ml.svg"]
```

Comment:
Sorry for drive-by comment; am looking at this for reference of another quickstart. Maybe a camera icon would be better since this plant watering one doesn't really fit?